apps/admin-gui/src/app/vos/components/member-group-list/member-group-list.component.html

```diff
@@ -1,59 +0,0 @@
-<div [class.hide-table]="exporting" [hidden]="dataSource.filteredData.length === 0 || groups.length === 0" class="card mt-2">
-  <div class="card-body">
-    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter"></perun-web-apps-table-options>
-    <div class="overflow-auto">
-      <table #exporter="matTableExporter" [dataSource]="dataSource" class="w-100" mat-table matSort matSortActive="id" matSortDirection="asc" matSortDisableClear matTableExporter>
-        <ng-container matColumnDef="select">
-          <th *matHeaderCellDef mat-header-cell>
-            <mat-checkbox (change)="$event ? masterToggle() : null"
-                          [aria-label]="checkboxLabel()"
-                          [checked]="selection.hasValue() && isAllSelected()"
-                          [indeterminate]="selection.hasValue() && !isAllSelected()"
-                          color="primary">
-            </mat-checkbox>
-          </th>
-          <td *matCellDef="let row" class="static-column-size" mat-cell>
-            <mat-checkbox (change)="$event ? selection.toggle(row) : null"
-                          (click)="$event.stopPropagation()"
-                          [aria-label]="checkboxLabel(row)"
-                          [checked]="selection.isSelected(row)"
-                          color="primary">
-            </mat-checkbox>
-          </td>
-        </ng-container>
-        <ng-container matColumnDef="id">
-          <th *matHeaderCellDef mat-header-cell mat-sort-header>{{'MEMBER_DETAIL.GROUPS.TABLE_GROUP_ID' | translate}}</th>
-          <td *matCellDef="let group" class="static-column-size" mat-cell>{{group.id}}</td>
-        </ng-container>
-        <ng-container matColumnDef="name">
-          <th *matHeaderCellDef mat-header-cell mat-sort-header>{{'MEMBER_DETAIL.GROUPS.TABLE_GROUP_NAME' | translate}}</th>
-          <td *matCellDef="let group" mat-cell>{{group.name}}</td>
```

Comment:
these tags are not used now(MEMBER_DETAIL.GROUPS.TABLE_GROUP_NAME), delete them in en.json